packages/edit-post/src/components/header/index.js

```diff
@@ -44,7 +44,7 @@ function Header( { setEntitiesSavedStatesCallback } ) {
 				select( editPostStore ).isFeatureActive( 'distractionFree' ) &&
 				isLargeViewport,
 		} ),
-		[]
+		[ isLargeViewport ]
```

Comment:
Question. Do we want the other selectors to be re-run when the viewport changes? If not should we decouple this selector and run separately?

Reply:
I think it's fine `isLargeViewport` is not going to change very often.